flink-runtime-web/src/test/java/org/apache/flink/runtime/webmonitor/WebRuntimeMonitorITCase.java

```diff
@@ -479,41 +479,6 @@ public void testNoCopyFromJar() throws Exception {
 		}
 	}
 
-	private WebRuntimeMonitor startWebRuntimeMonitor(
```

Comment:
This change belongs into the previous commit, i.e. `Create DispatcherRestEndpoint and integrate with Dispatcher`.

Reply:
True. Will remove this change from the PR.